dispatch.yaml

```diff
@@ -0,0 +1,4 @@
+dispatch:
+  # Send requests for exports to the cloud datastore admin service.
+  - url: "*/cloud-datastore-export/*"
```

Comment:
Maybe use underscores instead of hyphens, same as we do in other places.

Reply:
done